### PR TITLE
Disable enable zone pod cluster and host

### PR DESCRIPTION
**Pull request by pritisarap12**
Test cases for testing the behavior of resources running on
                zone, cluster, pod, host and admin/non-admin user after disabling the
                zone, cluster, pod, host respectively

### CODE REVIEW

**Comment by pritisarap12** [comment]
Disable enable zone pod cluster and host:
         --Added missing strings in codes.py

File: test/integration/component/maint/testpath_disable_enable_zone.py [hunk 1]
```diff
@@ -429,71 +380,1093 @@ def test_01_disable_enable_zone(self):
...
+    def tearDownClass(cls):
+        try:
+            clusterList = Cluster.list(cls.apiclient, id=cls.cluster.id)
+            if clusterList[0].allocationstate == DISABLED:
+                cmd = updateCluster.updateClusterCmd()
+                cmd.id = clusterList[0].id
+                cmd.allocationstate = ENABLED
+                cls.apiclient.updateCluster(cmd)
+
+            if clusterList[0].managedstate == "Unmanaged":
```
Comment (gauravaradhye) [inline test/integration/component/maint/testpath_disable_enable_zone.py]: Use lowercase for string comparison

File: test/integration/component/maint/testpath_disable_enable_zone.py [hunk 2]
```diff
@@ -429,71 +380,1093 @@ def test_01_disable_enable_zone(self):
...
+        self.apiclient.updateCluster(cmd)
+        clusterList = Cluster.list(self.apiclient, id=self.cluster.id)
+
+        self.assertEqual(clusterList[0].allocationstate,
+                         DISABLED,
+                         "Check if the cluster is in disabled state"
+                         )
+        # Verify the exsisting vms should be running
+        self.assertEqual(vm_user.state,
+                         RUNNING,
```
Comment (gauravaradhye) [inline test/integration/component/maint/testpath_disable_enable_zone.py]: Use lowercase

**Comment by pritisarap12** [comment]
Done with review changes.

**Pull request closed without merging**